tests/integration-service/integration.go

```diff
@@ -340,20 +340,29 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 				}, timeout, interval).Should(Succeed(), fmt.Sprintf("time out when trying to check if either the Snapshot %s/%s exists or if the Snapshot is marked as failed", testNamespace, snapshot.GetName()))
 			})
 
-			It("checks no Release CRs are created", func() {
-				releases, err := f.AsKubeAdmin.ReleaseController.GetReleases(testNamespace)
-				Expect(err).NotTo(HaveOccurred(), "Error when fetching the Releases")
-				Expect(releases.Items).To(BeEmpty(), "Expected no Release CRs to be present, but found some")
+			It("creates an snapshot of push event", func() {
+				sampleImage := "quay.io/redhat-appstudio/sample-image@sha256:841328df1b9f8c4087adbdcfec6cc99ac8308805dea83f6d415d6fb8d40227c1"
+				snapshot_push, err = f.AsKubeAdmin.IntegrationController.CreateSnapshotWithImage(componentName, applicationName, testNamespace, sampleImage)
+				Expect(err).ShouldNot(HaveOccurred())
 			})
 
-			It("checks no SnapshotEnvironmentBinding is created", func() {
-				seb, err := f.AsKubeAdmin.CommonController.GetSnapshotEnvironmentBinding(applicationName, testNamespace, env)
+			When("An snapshot of push event is created", func() {
```

Comment:
We'd better  include the Global Candidate List checking in line#368 to this "When" block although it's not added by STONEINT_468 task.

Reply:
updated.